numpy/core/src/umath/loops_arithmetic.dispatch.c.src

```diff
@@ -280,21 +280,30 @@ vsx4_simd_divide_contig_@sfx@(char **args, npy_intp len)
         npyv_b@len@ or         = npyv_or_@sfx@(ab_eq_cond, rem_zero);
         npyv_@sfx@ to_sub = npyv_select_@sfx@(or, vzero, vneg_one);
                       quo = npyv_add_@sfx@(quo, to_sub);
-        npyv_store_@sfx@(dst1, npyv_and_@sfx@(cvtozero, quo));
+                      // Divide by zero
+                      quo = npyv_select_@sfx@(bzero, vzero, quo);
+                      // Overflow
+                      quo = npyv_select_@sfx(overflow, vmin, quo);
```

Comment:
@ganesh-k13 : there is a missing `@` here:

```suggestion
                      quo = npyv_select_@sfx@(overflow, vmin, quo);
```


Applying the change above (the missing `@`), all tests pass on VSX4/Power10

I also see the warning message on VSX4/Power10 using the example shared above.
```
<stdin>:1: RuntimeWarning: overflow encountered in floor_divide
array([-2147483648, -2147483648, -2147483648, -2147483648, -2147483648,
       -2147483648, -2147483648, -2147483648, -2147483648, -2147483648],
      dtype=int32)
```

Reply:
Wooops :), thanks for having a look!  Planing on merging when tests are done (again).

Reply:
Wow, my bad! Thanks a lot!